feature-libs/user/profile/styles/my-account-v2/_my-account-v2-email.scss

```diff
@@ -0,0 +1,83 @@
+%cx-my-new-account-v2-email {
+  .email-enhancedUI-headertext {
+    font-size: $h3-font-size;
+    font-weight: var(--cx-font-weight-semi);
+    color: $text;
+    padding-bottom: 1.5rem;
+  }
+
+  .email-enhancedUI-editButton {
+    float: right;
```

Comment:
[MEDIUM]
Let's not use float, but flexbox in our CSS styles.
cc @developpeurweb 

Note: similar remark applies also to other SCSS files in this PR